instrumentation/instaecho/handler.go

```diff
@@ -0,0 +1,40 @@
+// (c) Copyright IBM Corp. 2021
+// (c) Copyright Instana Inc. 2016
+
+// +build go1.13
+
+package instaecho
+
+import (
+	"net/http"
+
+	instana "github.com/instana/go-sensor"
+	"github.com/labstack/echo/v4"
+)
+
+// AddMiddleware adds the tracing middleware to the list of Echo handlers.
+func AddMiddleware(sensor *instana.Sensor, engine *echo.Echo) {
```

Comment:
What is the purpose of this function? Shall we expose the `middleware()` and let users call `engine.Use()` themselves?

Reply:
It was done to have a similar interface as we already have for a gin framework. But I will agree, here it looks artificial. I will remove it. 
 

> Considering that our tracing middleware should be registered before any handlers, would it make sense instead of instaecho.AddMiddleware() provide instaecho.New() that would create a new Echo engine and register our middleware immediately?

I think you made a good suggestion with an `instaecho.New()` method. 👍 As far as there is only one constructor, let's wrap it.